src/cryptoadvance/specter/static/helper-modules/common.js

```diff
@@ -0,0 +1,64 @@
+function copyText(value, msg) {
```

Comment:
Can you explain what this function does (maybe better name and comments)? It is not clear to me.
![image](https://user-images.githubusercontent.com/60378539/194136542-36cf36ac-1554-44c7-9e4d-4c76f6fd12d0.png)

Reply:
Yeah, it is used for example here to copy an address:

<img width="660" alt="grafik" src="https://user-images.githubusercontent.com/70536101/194142179-e37f77b8-8620-4ee4-81dd-7493c9257aa1.png">

Reply:
Is there a reason this function is so complicated, and we dont user the Clipboard API https://www.w3docs.com/snippets/javascript/how-to-copy-the-text-to-the-clipboard-with-javascript.html#the-clipboard-api-6  ?   The compatibility for write seems ok  https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API